bin/mocha.js

```diff
@@ -109,7 +113,15 @@ if (mochaArgs['node-option'] || Object.keys(nodeArgs).length || hasInspect) {
   proc.on('exit', (code, signal) => {
     process.on('exit', () => {
       if (signal) {
-        process.kill(process.pid, signal);
+        const numericSignal =
+          typeof signal === 'string' ? os.constants.signals[signal] : signal;
```

Comment:
According to the documentation the `signal` argument of the `exit` event on `child_process.spawn()` is [always a string](https://nodejs.org/api/child_process.html#event-exit) – when is it ever something else?